### ISSUE TITLE
ShellExecuteEx failed: Environment variable not found.

### ISSUE THREAD

**Issue by aquaragon** [kocxyz/ServerDocker#4]
I get this error each time I try to spin up the docker image.

**Comment by Tandashi** [comment]
I guess you are missing an Environment variable then?

Without much information its hard to help.
What have you done so far? How does your setup look like? What commands are you trying to execute?

**Comment by aquaragon** [comment]
Yeah sorry about that. Ill try and provide as much as I can. At the moment I'm running into the error that entrypoint.sh can't cd into the directory I created. 
Am I mapping this correctly?
![Screenshot 2024-12-04 083448](https://github.com/user-attachments/assets/fdd63eff-70cc-42a5-b6d0-5d4e58010db2)
I created the folder KnockoutCityServer at /data/KnockoutCityServer

**Comment by Tandashi** [comment]
Have you downloaded the server data from Steam via SteamCMD into that Directory as well?


Download via SteamCMD using Docker:
`docker run --rm -it -v ./data:/data steamcmd/steamcmd:latest +@sSteamCmdForcePlatformType windows +force_install_dir /data +login <username> <password> +app_update 2915930 +quit`

Make sure to adjust the paths accordingly

**Comment by aquaragon** [comment]
Very sorry I did not see that until after. Thank you for the help.
